stable/graylog/templates/_helpers.tpl

```diff
@@ -42,13 +42,11 @@ Create the name of the service account to use
 Print Host URL
 */}}
 {{- define "graylog.url" -}}
-{{- if .Values.graylog.ingress.enabled }}
```

Comment:
Why URL is need when ingress is disabled?

Reply:
My thought here is to allow the service to expose its external uri when not using an ingress resource, like when it’s exposed via a node port or more close to our use case and internal load balancer via the service (gke specific).

I did not think a different value would be cleaner instead of reusing the ingress hosts one we have for simplicity but could easily be convinced otherwise.